### DISCUSSION TITLE
wingetcreate is disappointing

### DISCUSSION THREAD

**Discussion by OfficialEsco** [microsoft/winget-create#19] [Ideas]
wingetcreate is disappointing from what it sounded like it should have been..

**What i like**
* `wincreate new` will detect the InstallerType automatically 
* `wincreate new` will suggest pretty good metadata

**What i don't like**
* `wingetcreate` is not user-friendly 
* `wingetcreate` feels weird, why is it not `winget create`?
* `wingetcreate new` give way too few questions
* `wingetcreate update --id ` does... nothing? if the manifest does not use a dynamic url. if -v -u -s is not inputted it should prompt for the information.
* `wingetcreate update` does not prompt to update any of the information
* `wingetcreate update` should change the InstallerType and SilentSwitches if it detects a compatible InstallerType
* `wingetcreate update` does not update the ProductCode and will keep the previous one like wingetbot.

I will be sticking to [YamlCreate.ps1](https://github.com/microsoft/winget-pkgs/blob/master/Tools/YamlCreate.ps1)

**Comment by denelon** [comment]
@OfficialEsco lots to unpack here 😊.  Thanks for the feedback!

Re: "wingetcrate vs winget create".
One of the use cases is a standalone tool for CI scenarios to help automate publishing manifests. We do plan on integrating this tool so you can "winget create" with this tool. https://github.com/microsoft/winget-cli/issues/989

**Comment by denelon** [comment]
https://github.com/microsoft/winget-create/issues/20

`wingetcreate update` should change the InstallerType and SilentSwitches if it detects a compatible InstallerType

**Comment by denelon** [comment]
#22 

`wingetcreate update` does not update the ProductCode and will keep the previous one like wingetbot.

**Comment by denelon** [comment]
#23
`wingetcreate update` does not prompt to update any of the information

**Comment by denelon** [comment]
#24
`wingetcreate update --id` does... nothing? if the manifest does not use a dynamic url. if -v -u -s is not inputted it should prompt for the information.

**Comment by OfficialEsco** [comment]
Oke, `wingetcreate new` just added fuel to the fire by ruining manifests with tons of metadata

**Reply by denelon** [reply]
@OfficialEsco thanks for the notes on https://github.com/microsoft/winget-pkgs/issues/14621. Clearly we've got some work to do.
